pyuvdata/aipy_extracts.py

```diff
@@ -521,11 +652,23 @@ def write(self, preamble, data, flags=None):
         self.raw_write(preamble, data.astype(np.complex64), flags.astype(np.int32))
 
     def init_from_uv(self, uv, override={}, exclude=[]):
-        """Initialize header items and variables from another UV. Those in override
-        will be overwritten by override[k], and tracking will be turned off
-        (meaning they will not be updated in pipe()). Those in exclude are
-        omitted completely.
-
+        """
+        Initialize header items and variables from another UV.
+
+        Those in override will be overwritten by override[k], and tracking will
+        be turned off (meaning they will not be updated in pipe()). Those in
+        exclude are omitted completely.
+
+        TODO: check that defaults work properly. I usually use None in this context
```

Comment:
Do we want to leave a TODO in the code? Can we make this an issue instead?
The sentence "I usually use None in this context" is also ambiguous and I'm not sure how the user is supposed to use that.

Reply:
you're right, this isn't very helpful. I just made an issue, I'll take the TODO out of the code now.